core/src/main/java/hudson/ExtensionListView.java

```diff
@@ -107,12 +107,12 @@ public boolean remove(Object o) {
             }
 
             @Override
-            public void add(T t) {
+            synchronized public void add(T t) {
```

Comment:
Typical ordering is `public synchronized void`

Reply:
ok, I'll switch the order

Reply:
I did a `sed -i` and also flipped one existing case of `synchronized public` (there are now 225 cases of `public synchronized`, so you are certainly right that it is typical!).